rental.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require './person'
+require './book'
+
+# Class that represents a rental
+class Rental
+  attr_reader :person, :book
+
+  def initialize(person, book)
+    @person = person
+    @book = book
+    person.add_rental(self)
+    book.add_rental(self)
+  end
```

Comment:
- [ ] Dear @dfsalam, while creating a rental we need to set the date for the rental happens, Please make sure to have a rental variable that gen initialized while initializing the new rental, Also, please make sure to have the `setters` and `getters` for the `@date` instance variable.